community-operators/radanalytics-spark/sparkclusteroperator.0.3.4.clusterserviceversion.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sparkoperator.v0.3.4
+  namespace: openshift-operators
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "radanalytics.io/v1",
+          "kind": "sparkcluster",
+          "metadata": {
+            "name": "my-spark-cluster"
+          },
+          "spec": {
+            "worker": {
+              "instances": "2"
+            },
+            "master": {
+              "instances": "1"
+            }
+          }
+        },
+        {
+          "apiVersion": "radanalytics.io/v1",
+          "kind": "sparkapplication",
+          "metadata": {
+            "name": "my-spark-app"
+          },
+          "spec": {
+            "mainApplicationFile": "local:///opt/spark/examples/jars/spark-examples_2.11-2.3.0.jar",
+            "mainClass": "org.apache.spark.examples.SparkPi",
+            "driver": {
+              "cores": 0.2,
+              "coreLimit": "500m"
+            },
+            "executor": {
+              "instances": 2,
+              "cores": 1,
+              "coreLimit": "1000m"
+            }
+          }
+        }
+      ]
+    categories: apache spark, apache, spark
```

Comment:
An appropriate list of categories are being made to limit the number displayed in the UI. For Spark, I think "Big Data" would be appropriate.